druid/src/widget/svg.rs

```diff
@@ -24,8 +24,9 @@ use log::error;
 use usvg;
 
 use crate::{
-    kurbo::BezPath, BoxConstraints, Color, Data, Env, Event, EventCtx, LayoutCtx, PaintCtx, Point,
-    RenderContext, Size, UpdateCtx, Widget,
+    kurbo::{Affine, BezPath, Rect},
```

Comment:
more a note than anything: `Rect` and `Affine` are both reexported directly by druid, so you don't technically need to reach into kurbo to get them. `BezPath` isn't reexported because it isn't used in any of druid's API.

Reply:
I think I have applied this suggestion